externalpolicy/text_language/text_language_test.go

```diff
@@ -0,0 +1,119 @@
+// +build unit
+
+package text_language
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/open-horizon/anax/externalpolicy"
+)
+
+func Test_Validate_Succeed1(t *testing.T) {
+	// boolean, int, string
+	textConstraintLanguagePlugin := NewTextConstraintLanguagePlugin()
+	constraintStrings := []string{
+		"iame2edev == true && cpu == 3 NOT memory <= 32",
```

Comment:
question, so the NOT is treated as AND NOT?

Reply:
Discussed over slack and we decided to remove `NOT` and `^`